prow/tide/tide.go

```diff
@@ -65,6 +65,10 @@ type Controller struct {
 	kc     kubeClient
 	gc     *git.Client
 
+	// lastSyncStartTime is used to only list PRs that have changed since we last
+	// checked in order to make status context updates cheaper.
+	lastSyncStartTime time.Time
```

Comment:
how does this behave on the first iteration?

Reply:
The zero value for `time.Time` is `January 1, year 1, 00:00:00.000000000 UTC`: https://golang.org/pkg/time/#Time.
So long as we don't have any PRs last updated before that date we should be fine :smile:

Reply:
Right but does the github handle this correctly?

Reply:
Good call, of course Github barfs:
https://github.com/search?utf8=%E2%9C%93&q=repo%3Akubernetes%2Ftest-infra+updated%3A%3E%3D0001-01-01T00%3A00%3A00Z&type=

Reply:
Hmm, maybe we can pick a more recent minimum date. GitHub only dates back to 2008, what happens if we use the year 2000?

Reply:
I'm just going to omit the `update:>=...` tag from the search query on the first sync loop. The subsequent sync loops will all have a valid last sync time.

Reply:
SGTM